pkg/inventory/web/client.go

```diff
@@ -345,12 +346,28 @@ func (r *WatchReader) Repair() (status int, err error) {
 	return r.repair(r)
 }
 
+//
+// Reset logger.
+func (r *WatchReader) resetLog() {
+	r.log = logging.WithName(
+		"web|watch|reader",
+		"local",
+		r.webSocket.LocalAddr(),
+		"remote",
+		r.webSocket.RemoteAddr(),
+		"resource",
+		ref.ToKind(r.resource),
+		"watch",
+		r.id)
+}
+
```

Comment:
These WatchReader logger changes just improve logging.  This was helpful while troubleshooting other issues.